### ISSUE TITLE
first() returns an array

### ISSUE THREAD

**Issue by eleftrik** [yajra/laravel-oci8#78]
I just updated to 2.4.0.
Before the update, first() method returned an object, now it returns an array.
Thank you

**Comment by yajra** [comment]
@eleftrik, Yes, I did made some adjustment on this [commit](https://github.com/yajra/laravel-oci8/commit/dd9d84138e7904a5adcc610859b4864b3d3b7a5b). Did I break your code? I think it's the appropriate way to return an array when using query builder and object using eloquent.

```php
dd(App\Models\User::first()); // return eloquent object
// returns an array. Used to be returned as stdClass so when we return the response directly, 
// UnexpectedValueException is thrown.
dd(DB::table('users')->first()); 
```

**Comment by yajra** [comment]
Hmmm, I think I made a mistake here. Reviewing Laravel specs...

**Comment by yajra** [comment]
I reverted the changes I've made. Mistakes on my part. Sorry for the inconvenience. Fixed on v2.4.1.

**Comment by eleftrik** [comment]
Thank you @yajra!
